pandas/core/generic.py

```diff
@@ -7807,6 +7807,66 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         is not realigned. That is, use freq if you would like to extend the
         index when shifting and preserve the original data.
 
+        Examples
+        --------
+        Compute the difference between a column in a dataframe and
+        its shifted version
+
+        >>> data = pd.DataFrame({'mydate': [pd.to_datetime('2016-06-06'),
+        ...                                 pd.to_datetime('2016-06-08'),
+        ...                                 pd.to_datetime('2016-06-09'),
+        ...                                 pd.to_datetime('2016-06-10'),
+        ...                                 pd.to_datetime('2016-06-12'),
+        ...                                 pd.to_datetime('2016-06-13')],
+        ...                     'myvalue': [1, 2, 3, 4, 5, 6],
+        ...                     'group': ['A', 'A', 'A', 'B', 'B', 'B']})
+
+        >>> data.set_index('mydate', inplace=True)
+        >>> data
+                        myvalue group
+            mydate
```

Comment:
Is this indented too far? I thought the leftmost output should be directly below the leftmost `>`.

Reply:
noted